target/linux/mediatek/dts/mt7986a-mercusys-mr90x-v1.dts

```diff
@@ -0,0 +1,268 @@
+// SPDX-License-Identifier: (GL-2.0 OR MIT)
+
+/dts-v1/;
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
```

Comment:
No use here (dt-bindings/leds/common.h).

Reply:
Thanks. Line is dropped https://github.com/openwrt/openwrt/commit/e570c3390cce795b398a32d35e0c55eedd739928.